packages/core/src/tools/instrumentMethod.spec.ts

```diff
@@ -54,6 +54,17 @@ describe('instrumentMethod', () => {
     expect(instrumentationSpy).toHaveBeenCalled()
   })
 
+  it('handles undefined methods with third party wrappers after stop', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about moving this test to the `stop()` > `when the method has been instrumented by a third party` suite?

Reply:
Yeah, I was back and forth on which suite it belonged under. The condition is a result of stop being called, but the fix itself was more general (don't try to `apply` on a non-function), which is why I left it here.

Reply:
OK, since this test specifically target `stop` for third party, colocating it with the other related tests would be more appropriate.
If you see more general test cases related to this fix, please add them 🙂

Reply:
Thanks for the guidance, moved the test under `stop() -> third party`.